source/JointMilitarySymbologyLibraryCS/jmsml/test/Fields_ControlMeasuresAreas.csv

```diff
@@ -0,0 +1,26 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain
+identity,Integer,4,Standard Identity,NULLABLE,Identity
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,
+entity,Integer,4,Control Measures Area,NULLABLE,Control_Measure_Area_Entities
+echelon,Integer,4,echelon,NULLABLE,Echelons
+uniquedesignation,String,50,Unique Designation,NULLABLE,
+additionalinformation,String,50,Additional Information,NULLABLE,
+additionalinformation2,String,50,Additional Information 2,NULLABLE,
+hostile,String,3,Hostile (ENY),NULLABLE,
+type,String,50,Type of Equipment,NULLABLE,
```

Comment:
I didn't realize that these fields had different lengths based on the symbol set - is that intentional or can this be fixed/made more consistent in the standard (seems like more of an oversight to me)?

(Also when I looked - it showed "20" for Control Measures/Appendix H)
![image](https://cloud.githubusercontent.com/assets/3090809/9773465/63846d0c-5710-11e5-9b33-9ca617b8180e.png)

Also if there is an issue with "display length" versus recommended "database storage length" please clarify that when you document the GDB schema changes.